wdl2cwl/main.py

```diff
@@ -805,6 +852,23 @@ def get_cwl_task_inputs(
             )
 
         return inputs
+    def get_struct_inputs(self, members) -> List[cwl.CommandInputParameter]:
```

Comment:
Since type can accept other CommandInputParameter, the struct inputs is returned as a list of commandInputParameter for all the items it contains.